index.html

```diff
@@ -3352,15 +3359,52 @@ <h3 id="docconformance-attr">
           </tbody>
         </table>
       </section>
+      <section class="proposed addition">
+        <h2 id="docconformance-deprecated">
+          Requirements for deprecated ARIA role, state and property and attributes
+        </h2>
+        <p>
+          The ARIA Specification's <a data-cite="wai-aria-1.2#deprecated">Deprecated Requirements</a> section indicates that if an ARIA feature is marked as deprecated then authors are recommended to not use said feature for new content.
```

Comment:
coming in too late, but: "authors are recommended to not use" > "authors are advised not to use"